src/utils/media-utils.js

```diff
@@ -91,6 +91,21 @@ function getOrientation(file, callback) {
   reader.readAsArrayBuffer(file);
 }
 
+function getLatestMediaVersionOfSrc(src) {
```

Comment:
This feels a bit hairy. Also as written you would start back at version 1 after deleting all instances of a given src. Seems like this should be part of the resolver cache (or better yet part of resolving media on the backend, though that might be overkil)

Reply:
the problem here is the resolver cache is node-local, and non durable. so we have no durable, cross-node information about what the last-requested version is for a URL. without adding the durability, it means that information is lost when the last instance of an object is removed.

this behavior I wrote here seems better than *always* starting at version 1, since it's often the case you want to do a simple operation like clone and have it do what you expect, instead of handing you an old version.

so if we wanted to fix this, we'd need to start persisting the resolver cache to the database, which seems like a bad idea. it also would be a different cache than what we have now, which is `<url, version> -> resolved url`, and this cache is `url -> last version`